src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+
+        return color.toString();
```

Comment:
Use the `name()` method to get the String representation of the enum constant instead of `toString()`, which is the preferred way according to the checklist.